packit_service/models.py

```diff
@@ -701,21 +693,21 @@ def get_trigger_object(self) -> Optional[AbstractTriggerDbType]:
             )
 
     def __repr__(self):
-        return f"JobTriggerModel(type={self.type}, trigger_id={self.trigger_id})"
+        return f"JobTrigger(type={self.type}, trigger_id={self.trigger_id})"
 
 
-class RunModel(Base):
+class Pipeline(Base):
     """
     Represents one pipeline.
 
-    Connects JobTriggerModel (and triggers like PullRequestModel via that model) with
-    build/test models like  SRPMBuildModel, CoprBuildModel, KojiBuildModel, and TFTTestRunModel.
+    Connects JobTrigger (and triggers like PullRequest via that model) with
+    build/test models like  SRPMBuild, CoprBuildTarget, KojiBuildTarget, and TFTTestRunTarget.
 
     * One model of each build/test model can be connected.
-    * Each build/test model can be connected to multiple RunModels (e.g. on retrigger).
-    * Each RunModel has to be connected to exactly one JobTriggerModel.
-    * There can be multiple RunModels for one JobTriggerModel.
-      (e.g. For each push to PR, there will be new RunModel, but same JobTriggerModel.)
+    * Each build/test model can be connected to multiple Pipelines (e.g. on retrigger).
+    * Each Pipeline has to be connected to exactly one JobTrigger.
+    * There can be multiple Pipelines for one JobTrigger.
+      (e.g. For each push to PR, there will be new Pipeline, but same JobTrigger.)
     """
 
     __tablename__ = "runs"
```

Comment:
If you don't change the `__tablename__`, no change in a database is needed. Maybe, you need to rename only tables, run the alembic and then the class names so alembic can connect the models together.

In case of issues, I would ask @mfocko since he has done some renaming already:
https://github.com/packit/packit-service/blob/main/alembic/versions/70444197d206_rename_whitelist_to_allowlist.py

Reply:
Hm, no luck with having alembic do the changes yet, it either fails or just removes the existing table and creates a new one. I wrote the renaming manually but looking at Mato's commit, I also should rename the indexes. Will finish this migration manually

Reply:
I would probably make a list of `(original_name, new_name)` and run it through the for-loop, there are some differences here and there (e.g. table vs index renaming), but it should be much less code and easier to understand